torch/onnx/symbolic_opset17.py

```diff
@@ -54,3 +55,128 @@ def layer_norm(
         epsilon_f=eps,
         axis_i=axis,
     )
+
+
+@_onnx_symbolic("aten::stft")
+@symbolic_helper.parse_args("v", "i", "i", "i", "v", "b", "b", "b")
+@_beartype.beartype
+def stft(
+    g: jit_utils.GraphContext,
+    input: _C.Value,
+    n_fft: int,
+    hop_length: Optional[int] = None,
+    win_length: Optional[int] = None,
+    window: Optional[_C.Value] = None,
+    normalized: bool = False,
+    onesided: Optional[bool] = True,
+    return_complex: Optional[bool] = False,
+) -> _C.Value:
+    """Associates `torch.stft` with the `STFT` ONNX operator.
+    Note that torch.stft calls _VF.stft, without centering or padding options.
+    Hence, this function does not contain these two arguments.
+    See torch.stft source code for more info.
+
+    Parameters
+    ----------
+    g : jit_utils.GraphContext
+        Graph to write the ONNX representation into
+    input : _C.Value
+        Input tensor for the transformation
+    n_fft : int
+        FFT size
+    hop_length : int, optional
```

Comment:
note: no need to include types as they appear in the function definition already

Reply:
Done!